apps/src/code-studio/initApp/project.js

```diff
@@ -46,7 +46,6 @@ var events = {
   // Fired when run state changes or we enter/exit design mode
   appModeChanged: 'appModeChanged',
   appInitialized: 'appInitialized',
-  workspaceChange: 'workspaceChange',
```

Comment:
While `appModeChanged` and `appInitialized` events are triggered in our codebase, `workspaceChange` is not.

Removing unused event.

Reply:
Was this something that existed in cdo blockly maybe? Could we reproduce it with google blockly?

Reply:
Oh yeah! I found it in CDO Blockly - nice call! https://github.com/code-dot-org/blockly/blob/142ae2675f3c6a16c7e34379c46fa4028b5f7a37/core/ui/block_space/block_space.js#L831

Will push an update to replicate in Google Blockly - great idea!